tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -129,16 +131,19 @@ public static string GetNativeLoaderPath()
         {
             var monitoringHome = GetMonitoringHomePath();
 
-            string fileName = (EnvironmentTools.GetOS(), EnvironmentTools.GetPlatform()) switch
+            var (extension, dir) = (EnvironmentTools.GetOS(), EnvironmentTools.GetPlatform(), IsAlpine()) switch
             {
-                ("win", "X64")     => "Datadog.AutoInstrumentation.NativeLoader.x64.dll",
-                ("win", "X86")     => "Datadog.AutoInstrumentation.NativeLoader.x86.dll",
-                ("linux", "X64" or "Arm64") => "Datadog.Trace.ClrProfiler.Native.so",
-                ("osx", _)         => "Datadog.AutoInstrumentation.NativeLoader.dylib",
+                ("win", "X64", _) => ("dll", "win-x64"),
+                ("win", "X86", _) => ("dll", "win-x86"),
+                ("linux", "X64", false) => ("so", "linux-x64"),
+                ("linux", "X64", true) => ("so", "linux-musl-x64"),
+                ("linux", "Arm64", _) => ("so", "linux-arm64"),
+                ("osx", _, _) => ("dylib", "osx-x64"),
```

Comment:
I will have to change this once the support for both M1 (ARM64) and x64 arrives.

Reply:
Yup. There's a _lot_ of places we'll need to do that. Along with `win-arm64` 😅 I'd prefer to leave it as-is right now though, seeing as we don't support them yet